src/cli/config.ml

```diff
@@ -12,6 +12,9 @@ module CLT = Cmdliner.Term
 
 (** {3 Configuration type for common values} *)
 
+(** Output formats for the export command. *)
+type output = Lp | Dk
```

Comment:
Polymorphic variants may also be used, to avoid creating a new type (using `` `Lp`` instead of `Lp`)

Reply:
What would be the advantage?